python/ngen_conf/src/ngen/config/configurations.py

```diff
@@ -22,9 +22,9 @@ class Provider(str, Enum):
     file_pattern: Optional[Union[Path, str]]
     path: Path
     #reasonable? default
-    provider: Provider = Field(Provider.CSV)
+    provider: Forcing.Provider = Field(Provider.CSV)
```

Comment:
Pydantic... This is where this change set gets a little confusing. It comes down to the scoping of non-runtime type hints. So, in this case, `provider: Provider` causes an issue.

I think the most sane thing to do to avoid confusion and move `Provider` out of the class.